src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,23 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private double maxPushingCapacity;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(double maxPushingCapacity) {
```

Comment:
You should also specify an access modifier for the constructor that takes parameters. It's a good practice to be explicit about access levels.